app/theme_defaults/_sub_fields.twig

```diff
@@ -30,15 +30,9 @@
 
 {# SECTION 2: MACROS #}
 
-{# If 'allowtwig' is true for this field, we'll need to parse it as
-   Twig here. Note, that we're doing this inside a macro, so the
-   snippet has a limited scope. No global variables, etc. #}
-{% macro parsetextfield(key, value, contenttype) %}
-    {% if contenttype.fields[key].allowtwig is defined and contenttype.fields[key].allowtwig == true %}
-        {% set value = include(template_from_string(value), sandboxed = true) %}
-    {% endif %}
-
-    <div {% if key is not empty %}data-bolt-field="{{key}}"{% endif %}>{{ value }}</div>
+{# Output a text-like field with the wrapper for live editing. #}
+{% macro textfield(object, key) %}
+    <div data-bolt-field="{{key}}">{{ object.getDecodedValue(key) }}</div>
```

Comment:
OK, so expecting that I'd probably "Done a Gawain" and not communicated well enough, I went reading back though Twig internals to find the [part of the code that does to UPPER/lower case & `is`/`get` handling](https://github.com/twigphp/Twig/blob/1.x/lib/Twig/Template.php#L616-L657)

…then the :sweat_drops: :fish: … this works exactly the same:

```twig
<div data-bolt-field="{{key}}">{{ attribute(object, key) }}</div>
```

… and in turn no need for #6473 IMHO